.travis.yml

```diff
@@ -3,11 +3,14 @@ os:
     - linux
 osx_image:  xcode12.2
 python:
-    - "2.7"
     - "3.7"
+    - "3.8"
+    - "3.9"
```

Comment:
This whole section appears to be completely ignored by the Travis CI. See also #10.

Reply:
Indeed it was - on Mac OS, the `python: x.y` command was ignored, and the system default (2.7) was used instead. Moreover, `python --version` always returns 2.7. This issue was fixed in d3bb2ce.